cross-project-tests/debuginfo-tests/dexter/dex/debugger/DAP.py

```diff
@@ -606,7 +693,13 @@ def _get_launch_params(self, cmdline):
         """ "Set the debugger-specific params used in a launch request."""
 
     def launch(self, cmdline):
-        assert len(self.file_to_bp.keys()) > 0
+        # FIXME: This should probably not a warning, not an assert.
```

Comment:
At the moment, the reason it's an assert is that IIRC Dexter should always have either added a breakpoint or exited early at this point - so at the very least it would be a `raise`, rather than a warning.

Reply:
You may have instructed dexter to set a breakpoint, but the debugger fail to bind any. In that case, dexter asserting isn't very user friendly... in any case, I've reworded the fixme (I still believe it should be fixed at some point).